dnnv/nn/converters/tensorflow.py

```diff
@@ -685,6 +707,21 @@ def pad_func(*inputs):
 
         return pad_func
 
+    def visit_ReduceL2(self, operation):
+        x_ = operation.x
+        if isinstance(x_, Operation):
+            x_ = self.visit(x_)
+        axes = operation.axes
```

Comment:
`axes` is an input, so it could be an operation. Could you add a check for that and also add it to the _concretize function call below?